radicale/auth/ldap.py

```diff
@@ -275,12 +276,12 @@ def _login3(self, login: str, password: str) -> str:
             tmp: list[str] = []
             gdns: list[str] = []
             """Let's collect the groups of the user."""
-            if self._ldap_groups_attr == "memberOf":
-                gdns = user_entry['attributes']['memberOf']
-            elif self._ldap_groups_attr == "member" or self._ldap_groups_attr == "uniqueMember":
+            if self._ldap_groups_attr:
+                gdns = user_entry['attributes'][self._ldap_groups_attr]
+            elif self._ldap_group_members_attr:
                 conn.search(
```

Comment:
You may want to consider doing this search as the same user having done the user search, and not as the user found.
I am positively aware of LDAP setups where users are not allowed to search for anything apart from parts of their own entry. In such a case this search - when attempted as the user found (i.e. after checking the password using the bind() as user - will not yield results / fail.